src/test/java/api/support/builders/FeeFineOwnerBuilder.java

```diff
@@ -0,0 +1,71 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.List;
+import java.util.UUID;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineOwnerBuilder extends JsonBuilder implements Builder {
+  private UUID id;
+  private String owner;
+  private String desc;
+  private List<JsonObject> servicePointOwner;
+  private String defaultChargeNoticeId;
+  private String defaultActionNoticeId;
```

Comment:
Can these be final?

Reply:
Again, only if I initialize all of them with null in the no-argument constructor.

Reply:
If the sensible defaults for the builder are all null, then that is what I think should be done

Reply:
Done.